prow/flagutil/github.go

```diff
@@ -61,11 +63,19 @@ func (o *GitHubOptions) addFlags(wantDefaultGitHubTokenPath bool, fs *flag.FlagS
 // Validate validates GitHub options.
 func (o *GitHubOptions) Validate(dryRun bool) error {
 	for _, uri := range o.endpoint.Strings() {
-		if _, err := url.ParseRequestURI(uri); err != nil {
+		if uri == "" {
+			uri = github.DefaultAPIEndpoint
+		} else if _, err := url.ParseRequestURI(uri); err != nil {
 			return fmt.Errorf("invalid -github-endpoint URI: %q", uri)
 		}
 	}
 
+	if o.graphqlEndpoint == "" {
+		o.graphqlEndpoint = github.DefaultGraphQLEndpoint
```

Comment:
Defaulting is already handled as above.

Reply:
Same here, I am explicitly defaulting here, as some individual testing implementations don't run `addFlags`, so this value is empty for those tests.